cms/templates/js/content-group-details.underscore

```diff
@@ -41,17 +41,18 @@
 <% if (showContentGroupUsages) { %>
   <div class="collection-references wrapper-group-configuration-usages">
     <% if (!_.isEmpty(usage)) { %>
-      <h4 class="intro group-configuration-usage-text"><%= gettext('This content group is used in:') %></h4>
+      <h4 class="intro group-configuration-usage-text"><%- gettext('This content group is used in:') %></h4>
       <ol class="usage group-configuration-usage">
         <% _.each(usage, function(unit) { %>
           <li class="usage-unit group-configuration-usage-unit">
-            <p><a href=<%= unit.url %> ><%= unit.label %></a></p>
+            <p><a href=<%= unit.url %> ><%- unit.label %></a></p>
```

Comment:
@cahrens @efischer19 It is so simple for me to doubt earlier findings.  According to the [Java library documentation from oWASP](http://owasp-java-encoder.googlecode.com/svn/tags/1.1/core/apidocs/org/owasp/encoder/Encode.html#forUri(java.lang.String)), I'm pretty sure this can and should be HTML-escaped.

Here is the relevant comment from their code:

> This encoding is not intended to be used standalone. The output should be encoded to the target context. For example: `<a href="<%=Encode.forHtmlAttribute(Encode.forUri(uri))%>">...</a>`. (Note, the single-quote character (') is not encoded.)